crates/bevy_render/src/gpu_readback.rs

```diff
@@ -77,7 +77,10 @@ impl Plugin for GpuReadbackPlugin {
 #[derive(Component, ExtractComponent, Clone, Debug)]
 pub enum Readback {
     Texture(Handle<Image>),
-    Buffer(Handle<ShaderStorageBuffer>),
+    Buffer {
+        buffer: Handle<ShaderStorageBuffer>,
+        start_and_size: Option<(u64, u64)>,
```

Comment:
I think I'd prefer if this used `start_offset_and_size` but not a blocker

Reply:
Changed in https://github.com/bevyengine/bevy/pull/20133/commits/1f3cd9d4c7dcc2f5a0209c7173273defc19b53af